test/unit/sdk/exporters/collector_exporter_test.dart

```diff
@@ -59,7 +60,10 @@ void main() {
         resource,
         instrumentationLibrary,
         limits: limits,
-        attributes: [api.Attribute.fromBoolean('bool', true)],
+        attributes: [
+          api.Attribute.fromBoolean('bool', true),
+          api.Attribute.fromInt('int', 1),
+        ],
```

Comment:
is this covering all the types exposed by the sdk api?

Reply:
For sure it is not 🙃 
I aimed to fix the issue we are facing, but there a lot of things to be improved there

Reply:
I think this is critical, probably someone else will break this again, since the API allows that
WDYT @rafaelzlisboa ?